gateway/src/test/java/io/camunda/zeebe/gateway/interceptors/impl/IdentityInterceptorTest.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.interceptors.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.camunda.identity.sdk.Identity;
+import io.camunda.identity.sdk.authentication.exception.TokenVerificationException;
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
+import io.grpc.MethodDescriptor;
+import io.grpc.ServerCallHandler;
+import io.grpc.Status;
+import io.grpc.internal.NoopServerCall;
+import java.util.concurrent.atomic.AtomicReference;
+import org.junit.jupiter.api.Test;
+
+public class IdentityInterceptorTest {
+
+  @Test
+  public void missingTokenIsRejected() {
+    // given
+    final Identity identityMock = mock(Identity.class);
```

Comment:
:question: Maybe I'm dumb, but I guess here we don't need a mock? :thinking:

Reply:
I went for a Mock to not having to care about providing the needed config to create a real identity object.